src/Sylius/Bundle/AttributeBundle/Resources/config/services.xml

```diff
@@ -25,6 +30,19 @@
             <tag name="doctrine.event_subscriber" />
             <argument>%sylius.attribute.subjects%</argument>
         </service>
+
+        <service id="sylius.registry.attribute_type" class="%sylius.registry.attribute_type.class%">
+            <argument>Sylius\Component\Attribute\AttributeType\AttributeTypeInterface</argument>
```

Comment:
I don't think we will change that often nor we want to allow other's to change that. I'd leave it for now, if we think it is a good idea - then let's do it for all registries. Out of scope of this PR.